tests/cases/04_indexes.ts

```diff
@@ -14,7 +14,7 @@ export default function indexesTests() {
     assertEquals(
       res,
       {
-        createdCollectionAutomatically: false,
+        createdCollectionAutomatically: true,
```

Comment:
`createdCollectionAutomatically` is true because run `await db.collection("mongo_test_users").drop();` at the end of `03_curd.ts`

Ref: https://docs.mongodb.com/manual/reference/command/createIndexes/#output